### DISCUSSION TITLE
not able to get image during helm install of srsran

### DISCUSSION THREAD

**Discussion by skirubak** [srsran/srsRAN_Project#650] [General Help]
We are doing helm install.

helm install srsran-project srsran/srsran-cu-du --values=srsranValues.yaml -n srsran --create-namespace

We are getting image pull error.

  Warning  FailedToRetrieveImagePullSecret  4m31s (x18272 over 2d18h)  kubelet  Unable to retrieve some image pull secrets (regcred); attempting to pull the image may not succeed.

**Comment by skirubak** [comment]
We are also getting below error for linuxptp

tpdadmin@ashoktestserver:~$ kubectl logs  linuxptp-chart-dw4sx -n linuxptp 
Defaulted container "linuxptp-ptp4l" out of: linuxptp-ptp4l, linuxptp-phc2sys
ptp4l[5881143.270]: ioctl SIOCETHTOOL failed: No such device
ptp4l[5881143.278]: ioctl SIOCETHTOOL failed: No such device
ptp4l[5881143.278]: PTP device not specified and automatic determination is not supported. Please specify PTP device.
failed to create a clock
tpdadmin@ashoktestserver:~$

**Comment by brendan-mcauliffe** [comment]
@ninjab3s, could you take a look at this.

**Comment by ninjab3s** [comment]
Ill have a look, thanks @brendan-mcauliffe!

@skirubak the issue is that the container repo/image that you want to use cannot be accessed. The images that are currently online in our docker repo are not up to date. Please build the docker images using the supplied docker files in the repo and upload them to your own registry. Change the values in the values.yaml and redeploy the chart. We are currently working on pipelines to export images for every release.

I am closing this here because its not the correct repo for it. Please open a new issue in https://github.com/srsran/srsRAN_Project_helm if the error persists.
